installation/scripts/testing-auditlog-temp.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+source ${ROOT_PATH}/kyma-scripts/testing-common.sh
+
+readonly TMP_DIR=$(mktemp -d)
+
+echo "ARTIFACTS: ${ARTIFACTS}"
+readonly JUNIT_REPORT_PATH="${ARTIFACTS:-${TMP_DIR}}/junit_compass_octopus-test-suite.xml"
+
+suiteName="testsuite-all"
+echo "----------------------------"
+echo "- Testing Compass..."
+echo "----------------------------"
+
+kc="kubectl $(context_arg)"
+
+${kc} get clustertestsuites.testing.kyma-project.io > /dev/null 2>&1
+if [[ $? -eq 1 ]]
+then
+   echo "ERROR: script requires ClusterTestSuite CRD"
+   exit 1
+fi
+
+matchNames=$(cat <<-END
+    matchNames:
```

Comment:
Should you add  new matchNames: `compass-director-external-services-integration`?

Reply:
I didn't add it yet, because there is no such test. I think the test author should include it